src/CMakeLists.txt

```diff
@@ -212,14 +218,15 @@ target_include_directories(viamcpp
     "$<INSTALL_INTERFACE:${VIAMCPPSDK_PROTOBUF_INCLUDE_DIRS}>"
 )
 
-# This shouldn't be necessary, but we need it so that the installation
-# path of the C++ SDK is added to the link line of consumers to that
-# `-lviam_rust_utils` can be meaningfully resolved.
-#
-# TODO: Remove this when we can import `viam_rust_utils` as a real
-# imported target with its own properties.
 target_link_directories(viamcpp
-  INTERFACE
+  PUBLIC
```

Comment:
This was actually a build system bug! It took me quite a while to figure out why `libviamcpp.so` wasn't ending up with an RPATH that included the paths of its dependent runtime libraries. But I had marked those link diretories as 'interface', meaning that only things which depended on `libviamcpp.so` got them, but not `libviamcpp.so` itself. By switching to `public`, now `libviamcpp.so` gets them too, as it should.